### PR TITLE
added dexie, initialisation and initialised options

### PR DESCRIPTION
**Pull request by bdbch**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -20,6 +23,11 @@
       return {
         //
       }
+    },
+    created () {
+      db.settings.each((setting) => {
```
Comment (bdbch) [inline src/App.vue:28]: could probably be removed, kept in for testing

**Review by lostdesign — APPROVED** [review]
ok

**Review by redxtech — COMMENTED** [review]
this one is fine but in the future use your own fork

**Comment by bdbch** [comment]
@redxtech just to make this clear - why are we using forks if we're in the organisation anyway? This way it's way easier to allow other devs access to my branches without adding several origins.

Otherwise we wouldn't had the need to setup the organisation in the first place.

**Comment by redxtech** [comment]
yeah i guess using the forks here is a better idea generally for the people in the org

**Pull request merged**